package.json

```diff
@@ -31,7 +31,9 @@
     "format": "prettier --write \"**/*.{js,ts}\"",
     "format:check": "prettier --list-different \"**/*.{js,ts}\"",
     "lint": "tslint --project tsconfig.json --fix",
-    "lint:check": "tslint --project tsconfig.json"
+    "lint:check": "tslint --project tsconfig.json",
+    "docs:2": "mkdir -p docs/1-Guides && mkdir -p docs/2-API && cp documentation/*.md docs/1-Guides && api-extractor run --local && generate-ts-docs markdown -i docs/2-API -o docs/2-API && cp README.md docs/ && docsify-auto-sidebar",
+    "docs:2:publish":"gh-pages -d docs"
```

Comment:
Remove: already handled by my travis script - We can maybe add it later (another review and commit)